queue/queueservice.js

```diff
@@ -40,6 +40,20 @@ var validateTask = function(task) {
   assert(task.deadline instanceof Date, "Expected task.deadline");
 };
 
+/** Priority to constant for use in queue name (should be a string) */
+const PRIORITY_TO_CONSTANT = {
+  high:   '5',
+  normal: '1'
+};
+_.forIn(PRIORITY_TO_CONSTANT, v => assert(typeof v === 'string'));
+
+/** Priority in order of priority from high to low */
+const PRIORITIES = [
```

Comment:
Why not just use this both for the list of priorities and the constant they map to?